src/services/MeasureService.ts

```diff
@@ -39,4 +40,45 @@ export class MeasureService implements Service<fhir4.Measure> {
     }
     return result;
   }
+
+  /**
+   * result of sending a POST or GET request to {BASE_URL}/4_0_1/Measure/$package or {BASE_URL}/4_0_1/Measure/:id/$package
+   * creates a bundle of the measure (specified by parameters) and all dependent libraries
+   * supports parameters id and/or url + version (optional)
+   */
+  async package(args: RequestArgs, { req }: RequestCtx) {
+    logger.info(`${req.method} ${req.path}`);
+
+    const params = gatherParams(req.query, args.resource);
+    const id = args.id || params.id;
+    const url = params.url;
+    const version = params.version;
+
+    if (!id && !url) {
+      throw new BadRequestError('Must provide identifying information via either id or url parameters');
+    }
+
+    // query construction
+    const query: Filter<any> = {};
+    if (id) query.id = id;
+    if (url) query.url = url;
+    if (version) query.version = version;
+    const measure = await findResourcesWithQuery<fhir4.Measure>(query, 'Measure');
+    if (!measure || !(measure.length > 0)) {
+      throw new ResourceNotFoundError(
+        `No resource found in collection: Measure, with: ${Object.keys(query)
```

Comment:
This is actually a bit weird in a couple of places, where the colon comes after with instead of the thing being identified. Changed up across the board.